FirebaseSessions/generate_testapp.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+readonly DIR="$( git rev-parse --show-toplevel )"
+
+#
+# Use the following command in this comment if you're iterating on the
+# podspec. This will prevent settings from changing when you reload the project.
+#
+# pod gen FirebaseSessions.podspec --local-sources=./ --auto-open --platforms=ios,macos,tvos
+#
+cd $DIR/FirebaseSessions/Tests/TestApp
+pod install
```

Comment:
Would we have steps to bring in the secret file `GoogleServiceInfo.plist` before opening up the project?

Reply:
So I'm not sure we're able to do this. The password is stored in GitHub actions and passed in as a variable to our yaml file https://github.com/firebase/firebase-ios-sdk/pull/10396/files#diff-09baab8602f4dabcc3ffaa06efbdf84abaecd1f2bed8523fe833b189308172a3R42

Reply:
@paulb777 Is there a way to make it happen in our shell scripts for the project generation?

Reply:
A Googler only script could copy it from its CitC location

Reply:
Added a script
<img width="1024" alt="Screen Shot 2022-11-01 at 9 48 31 AM" src="https://user-images.githubusercontent.com/555046/199248687-d5397af4-ad47-4cfc-8724-d95a31654f42.png">

Reply:
Awesome. This is great! Thanks for doing this @samedson . Thanks @paulb777 .